zest/releaser/tests/bzr.txt

```diff
@@ -58,9 +58,11 @@ Commit it:
     >>> cmd
     'bzr commit -v -m "small tweak"'
     >>> print(system(cmd))
-    Committing to: TESTTEMP/tha.example-bzr/
-    modified setup.py
-    Committed revision 2.
+    RED Committing to: TESTTEMP/tha.example-bzr/
```

Comment:
Using a red color here makes it seem like something went wrong. Wouldn't lightblue or yellow or so be more appropriate?